File: reef_manager/example_job_templates/hello_world.rs
pub fn run(dataset: &[u8]) -> impl Into<ReefOutput> {
    let msg = "Hello World!";

    // Manual log invocation and println are the same for reef Rust.
    reef::reef_log(msg);

    // You can use format strings here.
    println!("{msg} {}", 2);
}

File: reef_manager/example_job_templates/hello_world.c
#include "reef.h"

void greet_n_times(int n) {
  for (int i = 0; i < n; i++) {
    reef_puts("Hello World!");
    reef_sleep(0.2);
    reef_progress((float) i / (float) n);
  }
}

void run(uint8_t *dataset, size_t ds_len) {
  // Write your main code here, as if it
  // were the `main` function.
  greet_n_times(10);
}
